### PR TITLE
✨ New Feature - Added Table Headers + Validation

### LINKED ISSUE #939: ✨ T&Cs - Add subheadings to the table
cc @pierssinclairssw @UlyssesMaclaren

https://www.ssw.com.au/terms-and-conditions

image

**Figure: ❌ Too many rows makes this table hard to read**

image
**Figure: ✅ Subheadings seperate sections**

```[tasklist]
### Tasks
- [x] Allow users to add subheadings to categorise roles in the table using Tina
- [x] Add subheadings - Developers, SysAdmins, Designers, TV + Marketing
- [x] Seperate Data Engineers and Software Engineers
- [x] Checked by Brady then Piers
- [ ] Complete Done Video v2 for combined #939 and #933
```

### PR DESCRIPTION
**Pull request by Harry-Ross**
* Added table subheadings 
* Added validation for if you add too many cells in one row (matched to how many table headers there are)

Fixes #939

Affected routes:

- `/terms-and-conditions`
- `/consulting/software-audit`

image
**v1 subheadings (red highlight)**

image
**v2 subheadings (text)**

### CODE REVIEW

File: content/pages/terms-and-conditions.mdx
Comment (bradystroud) [inline content/pages/terms-and-conditions.mdx]: Clean up keyboard spam :)

**Review by bradystroud — APPROVED** [review]
LGTM - Before you merge, can you get written approval from @UlyssesMaclaren ?
I know he has already verbally approved it.

A comment or approval on this PR is enough

File: content/pages/terms-and-conditions.mdx
Comment (bradystroud) [inline content/pages/terms-and-conditions.mdx]:
```suggestion
    { cells: [{ cellValue: "SysAdmins" }], isHeader: true },
```

**Review by UlyssesMaclaren — APPROVED** [review]

**Pull request merged**
